packages/basti/src/cli/commands/common/to-elasticache-choises.ts

```diff
@@ -1,16 +1,41 @@
 import inquirer from 'inquirer';
 
+import {
+  parseElasticacheMemcachedCacheNode,
+  parseMemcachedCacheClusterResponse,
+} from '#src/aws/elasticache/parse-elasticache-memcached-response.js';
 import {
   parseNodeGroupMemberResponse,
   parseNodeGroupResponse,
-} from '#src/aws/elasticache/parse-elasticache-response.js';
-import type { AwsElasticacheGenericObject } from '#src/aws/elasticache/elasticache-types.js';
-import type { ElasticacheClusterTargetInput } from '#src/target/target-input.js';
+} from '#src/aws/elasticache/parse-elasticache-redis-response.js';
+import type {
+  AwsElasticacheGenericObject,
+  AwsElasticacheMemcachedCluster,
+} from '#src/aws/elasticache/elasticache-types.js';
+import type { ElasticacheRedisClusterTargetInput } from '#src/target/target-input.js';
 import { fmt } from '#src/common/fmt.js';
 
-import type { NodeGroup } from '@aws-sdk/client-elasticache';
+import type { CacheCluster, NodeGroup } from '@aws-sdk/client-elasticache';
 import type { DistinctChoice } from 'inquirer';
 
+export function toElasticacheChoices(
+  redisClusters: AwsElasticacheGenericObject[][],
+  cacheClusters: AwsElasticacheGenericObject[],
+  memcachedClusters: AwsElasticacheMemcachedCluster[],
+  memcachedRawClusters: CacheCluster[],
```

Comment:
Please, give meaningful names to these arguments (and to arguments of other functions in this file as well). They can't all be clusters